runtime/bcutil/ROMClassBuilder.cpp

```diff
@@ -248,7 +252,31 @@ ROMClassBuilder::buildROMClass(ROMClassCreationContext *context)
 	context->recordLoadEnd(result);
 	return result;
 }
+BuildResult
+ROMClassBuilder::injectInterfaces(ClassFileOracle *classFileOracle)
+{
+	PORT_ACCESS_FROM_PORT(_portLibrary);
+	if (NULL == _injectionInfo) {
+		_injectionInfo = (InjectionInfo *) j9mem_allocate_memory(sizeof(InjectionInfo), J9MEM_CATEGORY_CLASSES);
```

Comment:
i think you can avaoid the allocation all together by just declaring it as an inline struct